Wallet-extension.md (#277)
ElrondNetwork/docs.elrond.com: docs/wallet/wallet-extension.md

**To get started:** 

**Step 1:** With the [Maiar DeFi wallet extension](https://chrome.google.com/webstore/detail/maiar-defi-wallet/dngmlblcodfobpdpecaadgfbcggfjfnm) installed. Click on ‘’Import existing wallet”.

  ![maiar-defi-wallet-welcome-page](https://user-images.githubusercontent.com/52820835/151935557-3be3f085-40a3-415e-b8a2-afbee54e1313.PNG)
 
**Step 2:** Next, enter your 24-word secret phrase. You can either enter these phrases one at a time or you can simply paste in the phrase using the "_paste_" icon.

   ![importwallet](https://user-images.githubusercontent.com/52820835/151940645-b25dcee5-4a37-4950-80e8-434924f9f4d3.PNG)
  
**Step 3:** Enter in your wallet password and confirm this password. 

  ![password3](https://user-images.githubusercontent.com/52820835/151942033-04782d44-982d-44d4-b910-5890a7d256e7.PNG)
  
**Step 4:**  Completed! Your Maiar DeFi Wallet has been successfully imported and set to be used.

   ![successful](https://user-images.githubusercontent.com/52820835/151939047-e8b09e1f-48f7-456b-a6f6-be0a42f83e4e.PNG)
   

## Key features

### Send to a wallet

> Welcome to Maiar DeFi Wallet Extension! One of the key features of this extension is that it allows you to send funds from your wallet to another wallet. To use this feature, you will need to have some funds in your wallet before proceeding. 

Reviewer: The welcome message should appear earlier, or maybe in a different form. You probably wanted to signal that the "setup" process is now complete and the wallet can now be used, right?

How about something like "Now you have a wallet registered in the Maiar DeFi Wallet Extension and it's ready to use. Great! Here's what you can do with it." Such a phrase could appear immediately after `## Key features`, but before `### Send to wallet` (between the two headings, basically).

Was this the intended effect?

**To get started**

 **Step 1:** Go to the Maiar Wallet extension, enter your password and click on _“**Send**”_.
 
  ![wallet_extension_step9](https://user-images.githubusercontent.com/52820835/151948316-0d68ba0a-d850-48ed-ad48-b7fd32589708.PNG)

**Step 2:** Enter the address of the wallet you intend to send to and the amount. 

![maiar-defi-example-send](https://user-images.githubusercontent.com/52820835/151948719-753c26c1-ac36-43c0-a4e2-1719f7fe4139.PNG)

*(Optional)* **Step 3**: Enter the data. This is a description of the transaction or any information you wish to pass through. 

**Step 4:** Click on the _“Continue”_ button to complete the transaction.

### Lock/unlock

> After 60minutes of being inactive, the extension automatically locks itself. You can unlock it at any time using your password. In addition, you can lock the extension manually, by clicking the **“lock”** icon in the header. 

Reviewer: ```suggestion
After 60 minutes of being inactive, the extension automatically locks itself. You can unlock it at any time using your password. In addition, you can lock the extension manually, by clicking the **“lock”** icon in the header. 
```

You can now connect Maiar Exchange to the Maiar DeFi wallet in real-time. With this connection, you will be able to log in to the Maiar exchange using the Maiar DeFi wallet extension in a few steps.
Follow these steps to proceed:

**Step 1:** To get started, go to the [Maiar Exchange](https://maiar.exchange/) page on the right section of the page, click on the  _“connect”_ button.

![connect-maiar](https://user-images.githubusercontent.com/52820835/151407628-6e9d30ff-b0c6-456a-a485-a50e511c31e1.PNG)

**Step 2:** Select "***Maiar DeFi Wallet***" from the options displayed.

![connect-maiar2](https://user-images.githubusercontent.com/52820835/151407721-89f6a637-891b-4fa8-a161-38c3a287f28d.PNG)

**Step 3:** Lastly, enter your password and click on the wallet address you want to connect to. 

![connectmaiar3](https://user-images.githubusercontent.com/52820835/151407901-ca06d7ca-1223-4306-9b92-b44de4d6dc8d.PNG)

- In a split second, the Maiar Exchange home page automatically reloads. You’ll notice your account has been added to the right section of the page.

![connectmaiar4](https://user-images.githubusercontent.com/52820835/151418100-cbcef037-d0ec-4eeb-b279-d9df3b444896.PNG)

> Successful 🎉

Reviewer: ```suggestion
Success! 🎉
```

* You should receive a notification that the extension has been added successfully. 
  ![maiar-defi-install-pop-up](/wallet/wallet-extension/wallet_extension_step3.png)

### Set up Maiar DeFi wallet
* Once it has been successfully installed, click on the extension to get started.

* You will be presented with two options: you can either *Create new wallet* or *Import existing wallet.*

![maiar-defi-example-welcomepage](/wallet/wallet-extension/wallet_extension_step4.png)

### Create a new wallet

**Step 1:** To get started, first install the [Maiar DeFi wallet extension](https://chrome.google.com/webstore/detail/maiar-defi-wallet/dngmlblcodfobpdpecaadgfbcggfjfnm).

**Step 2:** Open up the extension and click on _‘’Create new wallet”_.

> **Step 3:** Next, a set of 24-secret phrases will be displayed. Safely backup these secret phrases. We strongly recommend you copy the phrases or write them down and store in a safe place like a password manager.  These phrases would be used to access your wallet and may not be retrieved.

Reviewer 2: ```suggestion
**Step 3:** Next, a secret phrase consisting of a set of 24 secret words will be displayed. Safely backup these secret words. We strongly recommend you write them down or copy and store them in a safe place like a password manager.  These secret words are the key to your wallet account and cannot be recovered if lost.
```

![maiar-defi-example-secretphrase](/wallet/wallet-extension/wallet_extension_step5.png)

**Step 4:** Before proceeding to the next step, confirm that you have safely stored your secret phrase.

> **Step 5:** For further verification, you will be prompted to input some of the secret phrases. 

Reviewer 2: ```suggestion
**Step 5:** For further verification, you will be prompted to input some of the secret words. 
```

![maiar-defi-example-authentication](/wallet/wallet-extension/wallet_extension_step6.png)

> **Step 6:** Create a password that will be used to access your wallet. Ensure you keep this password safe as it will be needed to access your wallet regularly and this password cannot be retrieved. 

Reviewer 2: ```suggestion
**Step 6:** Create a password that will be used to access the wallets stored in the Maiar DeFi wallet extension. Ensure you keep this password safe as it will be needed to access your wallets regularly. Please note that this password cannot be recovered if lost. 
```

![maiar-defi-example-password](/wallet/wallet-extension/wallet_extension_step7.png)

**Step 7:** Completed! Your Maiar DeFi Wallet has been successfully created and set to be used. 

![maiar-defi-example-successful](/wallet/wallet-extension/wallet_extension_step8.png)

### Import existing wallet

Do you already have a wallet?

> Then there is no need to create a new one. The Maiar Wallet Extension provides an option to import your existing wallet. However, to import an existing wallet, you must be able to access the **recovery (secret) phrase** of your wallet. 

Reviewer 2: ```suggestion
Then there is no need to create a new one. The Maiar Wallet Extension provides an option to import your existing wallet. However, to import an existing wallet you must have access to its **secret (recovery) phrase**.
```

> The Maiar wallet has a set of 24-secret phrases, which serve as your wallet’s key. Using your secret phrase to import an existing wallet does not affect your wallet, as all transaction history across all networks will be reflected in the wallet extension.

Reviewer 2: I don't understand what you want to describe here. The "import an existing wallet does not affect your wallet" part is confusing.

Reviewer: Use the term `words` when referring to the 24 elements of the secret phrase. Also, make sure to only use "secret phrase" when referring to the 24 words (e.g. not "key"). This is important for consistency.

The second sentence of this paragraph, while reassuring to the user, is not that necessary. If you want to assure the user that importing a wallet is safe, you could maybe say that "Using a secret phrase to import an existing wallet does not affect the wallet in any way." Referring to the transaction history may be too much here.

**To get started:** 

**Step 1:** With the [Maiar DeFi wallet extension](https://chrome.google.com/webstore/detail/maiar-defi-wallet/dngmlblcodfobpdpecaadgfbcggfjfnm) installed. Click on ‘’Import existing wallet”.

![maiar-defi-example-welcomepage](/wallet/wallet-extension/wallet_extension_step4.png)

> **Step 2:** Next, enter your 24-word secret phrase. You can either enter these phrases one at a time or you can simply paste in the phrase using the "_paste_" icon.

Reviewer 2: ```suggestion
**Step 2:** Next, enter your 24-word secret phrase. You can either enter these words one at a time or you can simply paste in the entire phrase using the "_paste_" icon.
```

> ![importwallet](https://user-images.githubusercontent.com/52820835/151404597-537e8639-e5e8-47a8-9b85-bcd4951bfd1e.PNG)

Reviewer 2: Careful with these links. All images should be stored locally on our site, like: /wallet/wallet-extension/wallet_extension_step4.png

**Step 3:** Enter in your wallet password and confirm this password. 

> ![password](https://user-images.githubusercontent.com/52820835/151404896-4f92dfe3-4a22-4b55-80d7-ea52b1312ba2.PNG)

Reviewer 2: Check link.

**Step 4:**  Completed! Your Maiar DeFi Wallet has been successfully imported and set to be used.

> ![successful1](https://user-images.githubusercontent.com/52820835/151405157-ffc74a16-9e1c-4ebf-9875-d3e78907727f.PNG)

Reviewer 2: check link

*(Optional)* **Step 3**: Enter the data. This is a description of the transaction or any information you wish to pass through. 

**Step 4:** Click on the _“Continue”_ button to complete the transaction.

### Lock/unlock

After 60minutes of being inactive, the extension automatically locks itself. You can unlock it at any time using your password. In addition, you can lock the extension manually, by clicking the **“lock”** icon in the header. 

### Deposit to a wallet

A deposit can be made to your wallet using the wallet extension. This feature allows you to share your QR code or wallet address to receive a token deposit. To get started:

- Open up your Maiar Wallet extension.

- Next, click on the _"*deposit*"_ and share your QR code or wallet address.
 
![maiar-defi-example-deposit](/wallet/wallet-extension/wallet_extension_step11.png)

### Transactions history

On the wallet extension dashboard, the wallet records all transactions sent and received in your wallet. If you are a new user, it says "*No transactions found for this wallet*" until you make your first transaction.

![maiar-defi-example-transaction](/wallet/wallet-extension/wallet_extension_step12.png)

### Networks

> In the settings section on your extension dashboard, you can connect to the different networks provided by Elrond, such as the mainnet, testnet, and devnet. These networks will be discussed further in this documentation.

Reviewer 2: ```suggestion
In the settings section on your extension dashboard, you can connect to the different networks provided by Elrond, such as the mainnet, testnet, and devnet.
```

Choose either of these networks.

*More information about these networks is contained subsequently in this documentation.*

## Connecting the Maiar DeFi Wallet to Maiar Exchange App 

You can now connect Maiar Exchange to the Maiar DeFi wallet in real-time. With this connection, you will be able to log in to the Maiar exchange using the Maiar DeFi wallet extension in a few steps.
Follow these steps to proceed:

**Step 1:** To get started, go to the [Maiar Exchange](https://maiar.exchange/) page on the right section of the page, click on the  _“connect”_ button.

> ![connect-maiar](https://user-images.githubusercontent.com/52820835/151407628-6e9d30ff-b0c6-456a-a485-a50e511c31e1.PNG)

Reviewer 2: Check link.

**Step 2:** Select "***Maiar DeFi Wallet***" from the options displayed.

> ![connect-maiar2](https://user-images.githubusercontent.com/52820835/151407721-89f6a637-891b-4fa8-a161-38c3a287f28d.PNG)

Reviewer 2: Check link.

**Step 3:** Lastly, enter your password and click on the wallet address you want to connect to. 

> ![connectmaiar3](https://user-images.githubusercontent.com/52820835/151407901-ca06d7ca-1223-4306-9b92-b44de4d6dc8d.PNG)

Reviewer 2: Check link.

- In a split second, the Maiar Exchange home page automatically reloads. You’ll notice your account has been added to the right section of the page.

> ![connectmaiar4](https://user-images.githubusercontent.com/52820835/151418100-cbcef037-d0ec-4eeb-b279-d9df3b444896.PNG)

Reviewer 2: Check link.
